src/redux/sagas/meta/index.js

```diff
@@ -2,8 +2,10 @@ import { takeEvery } from 'redux-saga/effects'
 import types from '../../constants/market'
 import getCCYFullNames from './get_ccy_full_names'
 import getPerpsNames from './get_perps_names'
+import getCCYsId from './get_CCYs_id'
```

Comment:
rename get_CCYs_id to `get_ccys_id` to match with existing file naming like get_ccy_full_names. please do the same for other filenames that includes CCY